src/renderer/sw_engine/tvgSwRenderer.cpp

```diff
@@ -668,7 +673,7 @@ Compositor* SwRenderer::target(const RenderRegion& region, ColorSpace cs)
     cmp->w = cmp->compositor->image.w;
     cmp->h = cmp->compositor->image.h;
 
-    rasterClear(cmp, x, y, w, h);
+    rasterClear(cmp, clearColor, x, y, w, h);
```

Comment:
this is an intermediate target clear, so it should come up with "0" value. (not clearColor)